### ISSUE TITLE
Inconsistent error messages when VM in unknown state

### ISSUE THREAD

**Issue by sed-i** [canonical/multipass#3659]
**What are you trying to do?**
I forgot I had a VPN on and tried to launch a VM.
The error messages were contradicting: `not running` vs `already running`.

```
$ multipass list
Name                    State             IPv4             Image
charm-dev               Unknown           --               Ubuntu 22.04 LTS

$ multipass shell charm-dev
shell failed: Cannot retrieve credentials in unknown state

$ multipass restart charm-dev
restart failed: instance "charm-dev" is not running

$ multipass start charm-dev
start failed: The following errors occurred:
Instance 'charm-dev' is already running, but in an unknown state

$ multipass stop charm-dev
(This worked)
```

**What's your proposed solution?**
- The error messages should not contradict each other.
- The error messages should hint towards disabling the vpn.
- The error messages should hint towards using `stop`.

**Comment by ricab** [comment]
Indeed...

**Comment by ricab** [comment]
Hi @roy9495, sure! How do you propose to fix it?

Thank you for your interest in Multipass!

**Comment by roy9495** [comment]
Hi, I will do it like this
~~~
$ multipass shell charm-dev
shell failed: Cannot retrieve credentials in unknown state
If using VPN, disconnect it and try again

$ multipass restart charm-dev
restart failed: instance "charm-dev" is not running
Disconnect VPN, or use `multipass stop charm-dev`

$ multipass start charm-dev
start failed: The following errors occurred:
Instance 'Charm-dev' is already running, but in an unknown state
Disconnect VPN, or use `multipass stop charm-dev`
~~~

**Comment by ricab** [comment]
Hi @roy9495, I understand what you are trying to do with respect to the VPN suggestion, but there are many other possible causes for the instance to get into the `unknown` state and we can't print them all. We are trying to improve our troubleshooting docs though.

On the other hand, your proposed output wouldn't fix the issue reported here: that one message says its running and another says it isn't.

**Comment by roy9495** [comment]
Then, should this be the approach
~~~
$ multipass shell charm-dev
shell failed: Cannot retrieve credentials in unknown state

$ multipass restart charm-dev
restart failed: Instance 'Charm-dev' is already running, but in an unknown state
Use `multipass stop charm-dev`

$ multipass start charm-dev
start failed: The following errors occurred:
Instance 'Charm-dev' is already running, but in an unknown state
Use `multipass stop charm-dev`
~~~

**Comment by ricab** [comment]
Fixed by #3670 / #3706
